Makefile

```diff
@@ -26,7 +26,13 @@ DOC_FILES := \
 	project.md \
 	media-types.md \
 	manifest.md \
-	serialization.md
+	descriptor.md \
+	image-layout.md \
+	serialization.md \
+	layer.md \
```

Comment:
This order seems a bit odd to me.  I'd rather build this up from the ground with something like:

* …
* media-types
* descriptor
* image-layout
* layer
* serialization
* manifest